pex/pex_builder.py

```diff
@@ -428,11 +428,12 @@ def freeze(self, bytecode_compile=True):
       self._precompile_source()
     self._frozen = True
 
-  def build(self, filename, bytecode_compile=True):
+  def build(self, filename, bytecode_compile=True, deterministic_timestamp=False):
```

Comment:
Should this default to `True`? Or are we assuming that folks using pex as a library would not want something like this to change?

John would likely know better, but: my feeling is that this is a reasonable behaviour to change in a `1.6.x` -> `1.7.x` bump.

Reply:
The tentative plan is to change the default in 1.70 to be True. For now, we want to keep behavior the same as before during the transition period.

Reply:
Got it.